### DISCUSSION TITLE
Plans to add more features (like SD UI)

### DISCUSSION THREAD

**Discussion by btrbrt** [azuritecoin/OnnxDiffusersUI#18] [Q&A]
Hi,
are there plans to add more features (like the https://github.com/AUTOMATIC1111/stable-diffusion-webui has)?
Some of the most important for me would be:
* Add Restore Faces
* Input Seed
* easy switch for models
If you want to further build the tool and need support from a design side I can offer to support you. (I'm a proffessional UX/UI Designer)

**Comment by azuritecoin** [comment]
Hey,
I don't see myself adding too many more features to the current onnxUI. I feel it's a wasted effort to duplicate every feature from AUTOMATIC1111 webUI when it could be better spent porting their webUI to AMD on Windows (whatever that would require). Also ROCm on Linux gives much better performance with the same AMD card.

The features that I'll try to add are:  
- easy model switching
- In-painting
- longer prompts and prompt wieghts

Input seed should be implemented already (bugged for img2img), though you will need to copy and paste manually.
